force-app/main/default/triggers/OpportunityTriggers.trigger

```diff
@@ -0,0 +1,38 @@
+trigger OpportunityTriggers on Opportunity (before update, before delete) {
+    if (trigger.isBefore) {    
+        if (trigger.isUpdate) {
+            for (Opportunity opp : trigger.new) {
+                if(opp.Amount < 5000 ){
+                    opp.addError('Opportunity amount must be greater than 5000');
+                }
+                        
+            }
+            Map<Id,Contact> contactsMap = new Map<Id,Contact>([SELECT Id, AccountId, Title FROM Contact WHERE Title = 'CEO']);
+            Map<Id,Id> accountToContactsMap = new Map<Id,Id>();
+            for (Id conId : contactsMap.keySet()) {
+                accountToContactsMap.put(contactsMap.get(conId).AccountId,conId);
+            }
+            List<Contact> primaryContacts = new List<Contact>();
+            for (Opportunity opp : trigger.new) {
+                if (accountToContactsMap.keySet().contains(opp.AccountId)) {
```

Comment:
You dont need to specify keySet() here, you can use contains() right on the map. Same thing.